test/soll-runtime-test/soll-runtime-test.py

```diff
@@ -94,14 +105,17 @@ def testing(testbase):
             print_green(f"testcase-{i}: PASS")
         else:
             print_red(f"testcase-{i}: FAIL")
+            
+        #print(f"expected: {tohex32(v)} \nresult  : {tohex32(execute_result)}")
+        
 
     return num_testcase, result
 
 
 if __name__ == '__main__':
     files = []
     files.extend(glob.glob('./libyul/*/*.yul'))
-    files.extend(glob.glob('./yul/*/*.yul'))
+    #files.extend(glob.glob('./yul/*/*.yul'))
```

Comment:
If you want to comment this out, please write some comments to specify why does this line keep.